admin/taxes.md

```diff
@@ -26,10 +26,13 @@ If you use Avalara or TaxCloud to find a matching rate, that rate will override
 
 ![](/assets/admin-dashboard-taxes-avalara.png "Avalara Configuration")
```

Comment:
This is minor, but I think this screenshot is from an older version. Should we change it to a newer version?